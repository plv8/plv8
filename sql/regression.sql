-- https://github.com/plv8/plv8/issues/527
DO $$
  plv8.prepare('SELECT * FROM (VALUES (1)) AS foo').cursor().fetch.apply(null)
$$ language plv8;
