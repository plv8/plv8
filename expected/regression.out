-- https://github.com/plv8/plv8/issues/527
DO $$
  plv8.prepare('SELECT * FROM (VALUES (1)) AS foo').cursor().fetch.apply(null)
$$ language plv8;
ERROR:  cannot find cursor
CONTEXT:  undefined() LINE 2:   plv8.prepare('SELECT * FROM (VALUES (1)) AS foo').cursor().fetch.apply(null)
